addons/fleet/models/fleet_vehicle_cost.py

```diff
@@ -95,7 +95,8 @@ def default_get(self, default_fields):
 
     name = fields.Text(compute='_compute_contract_name', store=True)
     active = fields.Boolean(default=True)
-    start_date = fields.Date('Contract Start Date', default=fields.Date.context_today, 
+    user_id = fields.Many2one('res.users', 'Responsible', default=lambda self: self.env.user, index=True)
```

Comment:
Don't forget to explain why adding this field in commit message. I guess we needed a user for the generated activity ?